README.md

```diff
@@ -33,14 +33,15 @@ Prerequisites
     $ cd libtock-c
     ```
 
-1. The main requirement to build the C applications in this repository
-   is a cross compiler for embedded targets. Currently, the only
-   supportetd target is ARM Cortex M, so you will need an
-   `arm-none-eabi` toolchain.
+1. The main requirement to build the C applications in this repository is having
+   cross compilers for embedded targets. You will need an `arm-none-eabi`
+   toolchain for Cortex-M targets, and `riscv64-unknown-elf` toolchain for
+   RISC-V targets.
 
    MacOS:
    ```
    $ brew tap ARMmbed/homebrew-formulae && brew update && brew install arm-none-eabi-gcc
+   $ brew install riscv-gnu-toolchain --with-multilib
```

Comment:
```shell
$ brew install riscv-gnu-toolchain --with-multilib
[...]
Error: invalid option: --with-multilib

$ brew --version
Homebrew 2.3.0
Homebrew/homebrew-core (git revision 91f59; last commit 2020-06-05)
Homebrew/homebrew-cask (git revision 20cbf; last commit 2020-06-05)
```

?

Reply:
Arg that should be

`brew tap riscv/riscv` then install.